example-dvc-experiments/code/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "example-dvc-experiments",
+    "build": {
+        "dockerfile": "Dockerfile"
+    }
+}
```

Comment:
newline

Reply:
https://www.loom.com/share/d78aac3bf8c1489eac79e1da7e0550d5 :)

Reply:
loom! new way to communicate, I like it (for real) :)

I meant something like this https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file ... it's not usually about language specifically, it's about in general having end of lines at the end for UNIX tools to be have properly - just a best practice I guess, that's why GH highlights these missing newlines

not a big deal, clearly :)

Reply:
I'm also in the _EOL at the EOF_ camp and I believe it's standard but these days standards are determined by the editors I think.